webpack/webpack.config.common.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+module.exports = {
+  entry: './src/index.jsx',
+  output: {
+    path: path.join(__dirname, '..', '/dist'),
+    filename: 'app.bundle.js',
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+        },
+      },
+      {
+        test: /\.css$/,
+        use: ['style-loader', MiniCssExtractPlugin.loader, 'css-loader'],
+      },
+    ],
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './public/index.html',
+      favicon: "./public/favicon.ico"
```

Comment:
prettier doesn't help ?)

Reply:
Thanks for your note. I work in the IntelliJ idea and it was strange that the rule does not apply.